### ISSUE TITLE
Laterals csv upload not picking up laterals

### ISSUE THREAD

**Issue by Jonasvsl** [nens/threedi-api-qgis-client#95]
* threedi-api-qgis-client version: 2..0.1
* Python version: 3
* Operating System: Windows

### Description
Using the API client, selecting the laterals option. Uploading the csv (see attachment). Then nothing happened. Cannot override a lateral or see a sign that laterals have been uploaded.
After starting the simulation laterals are not picked up or sent via the API.

I used this model:

https://api.3di.live/v3.0/threedimodels/2334/

Describe what you were trying to get done.
Tell us what happened, what went wrong, and what you expected to happen.

### What I Did

```
Paste the command(s) you ran and the output.
If there was a crash, please include the traceback here.
```

**Comment by DamnBack** [comment]
@Jonasvsl Please send me your CSV file.

**Comment by DamnBack** [comment]
Anyway should be fixed in 2.0.2 release.
